### PR TITLE
[MM-33341] Make use of static assets

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Most of the changes are straight forward. The only tricky thing is that `svg` can't be used for static assets right now due to an issue with the HTTP `content-type` header (https://community-daily.mattermost.com/core/pl/qrpzsmxz6tb98gibzgdop5fisw). Hence, I've converted the image to an `png` for now, but kept the existing `svg` around. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-33341

### CODE REVIEW

File: src/bindings/channel_header.ts [hunk 1]
```diff
@@ -35,12 +35,12 @@ const channelHeaderSubscribe = (): AppBinding => {
...
 };
 
-const channelHeaderConfig = (): AppBinding => {
+const channelHeaderConfig = (context: AppContext): AppBinding => {
     return {
         location: CommandLocations.Configure,
         label: 'Configure Zendesk',
         description: 'Open Create Zendesk Config Modal',
-        icon: ZDIcon,
+        icon: getStaticURL(context, 'zendesk.png'),
```
Comment (larkox) [inline src/bindings/channel_header.ts:43]: Why not keep ZDIcon as `zendesk.png`? (That way if we change back to svg we only have to change it in one place)

**Review by larkox — APPROVED** [review]
LGTM, just a small nit.

File: src/bindings/channel_header.ts (hunk 1 above)
Comment (hanzei) [reply src/bindings/channel_header.ts]: Good suggestion :+1:

File: src/restapi/mm_routes.ts [hunk 2]
```diff
@@ -40,6 +40,11 @@ router.post(Routes.App.CallPathTicketOpenForm + '/submit', fOpenCreateTicketForm
 router.post(Routes.App.CallPathTicketSubmitOrUpdateForm + '/form', fSubmitOrUpdateCreateTicketForm);
 router.post(Routes.App.CallPathTicketSubmitOrUpdateForm + '/submit', fSubmitOrUpdateCreateTicketSubmit);
 
+// static files
+const staticRouter = express.Router();
+staticRouter.use(express.static('static'));
+router.use('/static', staticRouter);
```
Comment (mickmister) [inline src/restapi/mm_routes.ts:46]: Very static 🐕  such wow

**Review by mickmister — APPROVED** [review]

**Review by jfrerich — APPROVED** [review]
Looks great! Thanks, @hanzei

**Pull request merged**